.gitignore

```
@@ -1,4 +1,5 @@
 /target
+/*/target/
```

These can be folded together.

```suggestion
**/target
```